packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -72,10 +78,6 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 				eventName: "fluid:telemetry:ContainerRuntime:ClosingSummarizerOnSummaryStale",
 				message: "Stopping fetch from storage",
 			},
-			{
-				eventName: "fluid:telemetry:SummarizerNode:refreshLatestSummary_cancel",
```

Comment:
So instead we'll see `refreshLatestSummary_end` events?

Reply:
Yep, updated